debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -364,21 +365,58 @@ public Lsn getFromLsn() {
     }
 
     public Set<SqlServerChangeTable> listOfChangeTables(String databaseName) throws SQLException {
-        return queryAndMap(replaceDatabaseNamePlaceholder(GET_LIST_OF_CDC_ENABLED_TABLES, databaseName), rs -> {
+        return listOfChangeTables(databaseName, Lsn.NULL, Lsn.NULL);
+    }
+
+    public Set<SqlServerChangeTable> listOfChangeTables(String databaseName, Lsn fromLsn, Lsn toLsn) throws SQLException {
```

Comment:
The `fromLsn` parameter seems not to be used anywhere?

Reply:
It was used up until we realized that the end LSN is always NULL (first commit). I removed it from here and one more place.